examples/example_uncertainty.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import pandas as pd
+
+import pastas as ps
+
+ps.set_log_level("ERROR")
+
+# read observations and create the time series model
+obs = pd.read_csv("data/head_nb1.csv", index_col=0, parse_dates=True,
+                  squeeze=True)
+ml = ps.Model(obs, name="groundwater head")
+
+# read weather data and create stressmodel
+rain = pd.read_csv("data/rain_nb1.csv", index_col=0, parse_dates=True,
+                   squeeze=True)
+evap = pd.read_csv("data/evap_nb1.csv", index_col=0, parse_dates=True,
+                   squeeze=True)
+sm = ps.RechargeModel(prec=rain, evap=evap, rfunc=ps.Exponential,
+                      recharge="Linear", name='recharge')
+ml.add_stressmodel(sm)
+
+# Solve
+ml.solve()
+#
+df = ml.fit.prediction_interval()
+inside = (obs > df.loc[obs.index, 0.025]) & (obs < df.loc[obs.index, 0.975])
+print('results of prediction_interval2')
```

Comment:
I assume prediction_interval2 has become the default.

Reply:
Good point. pred2 has been removed. I'll change this.